src/applications/gi/components/AccordionItem.jsx

```diff
@@ -38,11 +37,8 @@ export default function AccordionItem({
           aria-expanded={displayExpanded}
           aria-controls={id}
           onClick={toggle}
-          className={
-            environment.isProduction()
-              ? 'usa-accordion-button vads-u-border--2px vads-u-border-style--solid vads-u-border-color--gray-light vads-u-margin--0'
-              : 'usa-accordion-button vads-u-margin--0'
-          }
+          className="usa-accordion-button vads-u-border--2px vads-u-border-style--solid vads-u-border-color--gray-light vads-u-margin--0"
```

Comment:
(On line 40) Shouldn't the className be _'usa-accordion-button vads-u-margin--0'_, given that the flagged ternary expression passed that to staging and kept the longer styling string in production?

Reply:
You right, good catch.